documentation/asciidoc/topics/proc_creating_service_monitor.adoc

```diff
@@ -1,52 +1,26 @@
 [id='creating-service-monitor_{context}']
 = Creating a Prometheus Service Monitor
-Create a Prometheus `ServiceMonitor` that scrapes metrics from a {brandname} cluster.
+{ispn_operator} can create a Prometheus `ServiceMonitor` that scrapes metrics from a {brandname} cluster.
 
 .Prerequisites
 
-* Have an `oc` client.
 * Enable monitoring for user-defined projects on {ocp}.
 
 .Procedure
 
-. Create an authentication secret for the `operator` user.
+. Set `'true'` as the value for the `infinispan.org/monitoring` annotation in your `Infinispan` CR.
```

Comment:
Set `false` to disable? This is tricky as it's going to be platform dependent.

Reply:
we call out the fact that the instructions are specific to OCP. however this is an interesting point. @dmvolod can users explicitly set `false` to disable monitoring for Infinispan clusters?

Reply:
Correct, @oraNod explicitly setting `false` means disable even if ServiceMonitor CRD is present